screens/Playlist.js

```diff
@@ -32,26 +30,17 @@ const storePlaylistData = async (playlistID, data) => {
     JSON.stringify(data)
   ).catch((err) => console.error(err));
 };
-
 const getStoredPlaylist = (playlistID) => {
   return AsyncStorage.getItem(`@${playlistID}`).catch((err) =>
     console.error(err)
   );
 };
 
 const PlaylistScreen = ({ navigation, route }) => {
-  const {
-    playlistID,
-    playlistTitle,
-    playlistDescription,
-    playlistURI,
-  } = route.params;
-  console.log(route.params);
-  navigation.setOptions({ title: playlistTitle });
   const insets = useSafeArea();
-  const ref = React.useRef(null);
+  const { playlistID, playlistTitle, playlistURI } = route.params;
 
-  useScrollToTop(ref);
```

Comment:
we don't need to handle this for this screen. this is only used for the main screens that aren't children of a stack